scripts/cargo-test.sh

```diff
@@ -19,6 +19,12 @@ trap cleanup_handler INT QUIT TERM HUP EXIT
 export PATH=$PATH:${HOME}/.cargo/bin
 set -euxo pipefail
 
+# setup extra modules as we need rdma-rxe for rdma tests
+sudo apt -y install rdmacm-utils
```

Comment:
hmm you can't assume everyone and ci is using ubuntu.
I think we should leave this out, should be part of pre-requisites, just like hugepages and nvme-tcp

Reply:
removed installation of package and added warning if kernel modules aren't installed.